### PR TITLE
OD-223 [Fix]  Data source manager Settings tab are not displayed in component configurations

### PR DESCRIPTION
**Pull request by AndrRyaz**
@sofiiakvasnevska 
## Issue
OD-223 https://weboo.atlassian.net/jira/software/projects/OD/boards/9?selectedIssue=OD-223
## Description
Fix to change fo needed params in object
## Screenshots/screencasts
https://screencast-o-matic.com/watch/cYXeITsPxl
This change is fully backward compatible.
## Reviewers
@upplabs-alex-levchenko @YaroslavOvdii

### CODE REVIEW

File: js/src/interface.js [hunk 1]
```diff
@@ -285,7 +285,8 @@ let app = new Vue({
         data: dataSourceData,
         onEvent: (event, dataSource) => {
           if (event === 'dataSourceSelect') {
-            this.selectedDataSource = dataSource;
+            this.selectedDataSource.columns = dataSource.columns;
```
Comment (tonytlwu) [inline js/src/interface.js:288]: If both `this.selectedDataSource` and `dataSource` have the same attributes (`columns` and `id`), why doesn't the current code (below) work?

```js
            this.selectedDataSource = dataSource;
```

Comment (AndrRyaz) [reply js/src/interface.js:288]: @tonytlwu  because this.selectedDataSource contains more params than columns and id

Comment (tonytlwu) [reply js/src/interface.js:288]: @AndrRyaz Can you clarify what params `this.selectedDataSource` contains?

Comment (AndrRyaz) [reply js/src/interface.js:288]: Sure 
изображение

Comment (tonytlwu) [reply js/src/interface.js:288]: @AndrRyaz There are attributes in `this.selectedDataSource` where if you only update `id` and `column` then the data loses its integrity, e.g. if you update the data source ID but don't change `updatedAt` then the data is incorrect.

If you don't need any other data except for `id` and `columns`, then you should just assign `dataSource` to `this.selectedDataSource`.  If there's anything in `this.selectedDataSource` that you need, then they should be kept up to date when `id` and `columns` change.

Comment (AndrRyaz) [reply js/src/interface.js:288]: @tonytlwu  Sorry, you're right, it doesn't make sense, the data needs to be updated

**Pull request closed without merging**